packages/website/src/components/Playground.tsx

```diff
@@ -96,73 +112,92 @@ function Playground(): JSX.Element {
         config={state.tsconfig}
         onClose={updateModal}
       />
-      <div className={clsx(styles.options, 'thin-scrollbar')}>
-        <OptionsSelector
-          isLoading={isLoading}
-          state={state}
-          tsVersions={tsVersions}
-          setState={setState}
-        />
-      </div>
       <div className={styles.codeBlocks}>
-        <div className={clsx(styles.sourceCode)}>
-          {isLoading && <Loader />}
-          <EditorTabs
-            tabs={['code', 'tsconfig', 'eslintrc']}
-            activeTab={activeTab}
-            change={setTab}
-            showModal={(): void => setShowModal(activeTab)}
-          />
-          <div className={styles.tabCode}>
-            <EditorEmbed />
+        <SplitPane
```

Comment:
Bug report: the playground is meant to be responsive. On smaller/mobile viewports, the options become a side menu that overlays the screen on menu button click. The left pane should respect that mobile responsiveness. You can repro this at 175% zoom:

![Screenshot of the playground zoomed in 175% with a blank left column](https://user-images.githubusercontent.com/3335181/191343872-bb0f911e-e9ae-4c58-9234-b74cfe84d776.png)

Reply:
Fixed